airflow/gcp/hooks/bigquery.py

```diff
@@ -332,19 +332,13 @@ def create_empty_table(self,
         self.log.info('Creating Table %s:%s.%s',
                       project_id, dataset_id, table_id)
 
-        try:
-            self.service.tables().insert(
-                projectId=project_id,
-                datasetId=dataset_id,
-                body=table_resource).execute(num_retries=num_retries)
-
-            self.log.info('Table created successfully: %s:%s.%s',
-                          project_id, dataset_id, table_id)
+        self.service.tables().insert(
+            projectId=project_id,
+            datasetId=dataset_id,
+            body=table_resource).execute(num_retries=num_retries)
 
-        except HttpError as err:
-            raise AirflowException(
-                'BigQuery job failed. Error was: {}'.format(err.content)
-            )
+        self.log.info('Table created successfully: %s:%s.%s',
```

Comment:
I don't get what changed! We just moved it from hook to operator !!

Reply:
Nearly, in case of http 409 we do not rise exception, because resource already exists.

Reply:
👍